auth/auth.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require("jsonwebtoken");
+const { getUserById } = require("../controllers/users");
+const secretJwt = process.env.SECRET_JWT;
+
+const auth = async (req, res, next) => {
+  const token = req.headers.authorization;
```

Comment:
```suggestion
  const token = req.headers.authorization;
```
Bearer dfsdsk3l2l3dsdsfs'